pallets/cash/src/core.rs

```diff
@@ -335,15 +335,31 @@ pub fn apply_chain_event_internal<T: Config>(event: ChainLogEvent) -> Result<(),
 pub fn dispatch_notice_internal<T: Config>(
     chain_id: ChainId,
     recipient_opt: Option<ChainAccount>,
+    should_increment_era: bool,
     notice_fn: &dyn Fn(NoticeId, ChainHash) -> Result<Notice, Reason>,
 ) -> Result<(), Reason> {
     // XXX this cannot fail, should not return result
     let (latest_notice_id, parent_hash) =
         LatestNotice::get(chain_id).unwrap_or((NoticeId(0, 0), chain_id.zero_hash()));
-    let notice_id = latest_notice_id.seq();
+
+    let notice_id = if should_increment_era {
+        require!(
```

Comment:
So we return an error on the extrinsic if there are any pending era notices to be signed? Need to think about this, didn't we ideally want to continue accepting transactions, just not signing them yet?